norns/reference.md

```diff
@@ -110,6 +110,65 @@ function cleanup()
 end
 ```
 
+## setting minimum required version
+
+Each norns update brings new features, both for scripting and general usability. If you share a script which relies on the features of a particular norns update, you can specify a minimum version requirement which will display a message on the norns screen if the script is loaded on a previous version of the software.
+
+To specify a minimum version required, set `norns.version.required` to the **YYMMDD** value before the `init()` of your script, eg:
+
+```lua
```

Comment:
this is a cool example but way too long to simply demonstrate the min version feature--- just an init function would do

Reply:
lol totally right, got bloaaaaty